contrib/docker-ckan/dev.env.template

```diff
@@ -76,7 +76,8 @@ NGINX_SSLPORT=443
 
 # Extensions
 # Keep the datasetformchain plugin at the end as it depends on the order to function correctly
-CKAN__PLUGINS="envvars datasci_sharing image_view text_view recline_view pdf_view cloudstorage datastore datapusher sso saml2auth smdh smdh_dataset_form smdh_group_form taglist usertracking granularvisibility generalpublic privatedatasets datasetformchain"
+# 'smdh' plugin is moved to the second last position because some of the overriding templates in ckanext-smdh are not using the ckan_extends.
+CKAN__PLUGINS="envvars datasci_sharing image_view text_view recline_view pdf_view cloudstorage datastore datapusher sso saml2auth falkor smdh_dataset_form smdh_group_form taglist usertracking granularvisibility generalpublic privatedatasets smdh datasetformchain"
```

Comment:
Can you make sure that the only changed plugin in this list is the `smdh` plugin. This change adds the `falkor` plugin as well, which was removed recently. Same thing for the prod env file.